### PR TITLE
fix(utils): improve svg security

### PR DESCRIPTION
**Pull request by TremayneChrist**
Adds patch to strip event attributes from any SVG body

### CODE REVIEW

File: packages/utils/src/loader/cdn-loader.ts [hunk 1]
```diff
@@ -24,6 +25,22 @@ export class CdnLoader {
...
+   * Strips any event attributes which could be used to
+   * maliciously hijack the application.
+   * @param element Element to check
+   * @returns {void}
+   */
+  private stripUnsafeAttributes (element: SVGElement): void {
+    const attributes = element.getAttributeNames();
+    for (const attribute of attributes) {
+      // Remove event attributes e.g., `onclick`
+      if (attribute.startsWith('on')) {
```
Comment (wsuwt) [inline packages/utils/src/loader/cdn-loader.ts]: `IE11` does not support `String.prototype.startsWith()`.

Comment (goremikins) [reply packages/utils/src/loader/cdn-loader.ts]: It should come as part of the default polyfill and automatically added by babel

Comment (wsuwt) [reply packages/utils/src/loader/cdn-loader.ts]: But I think babel preset should be able to handle this in most scenarios.

**Comment by TremayneChrist** [comment]
I've removed the previous code duplication and shifted logic into a separate `SVGLoader`, keeping the separate Flag/Icon loaders for preload logic.

Using @goremikins suggestion of `responseXML`, fixes the final known security flaw.

**Review by goremikins — DISMISSED** [review]

**Review by wsuwt — APPROVED** [review]

**Comment by dtanp-rft** [comment]
Bangkok team has already tried to hack the code but nobody is succeeding. I think we can merge this.

**Pull request merged**